scripts/test/config/perf-cluster.yml

```diff
@@ -2,6 +2,8 @@
 ---
 apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
+iam:
```

Comment:
Is this more for security / improvement or is it required?

Reply:
I wanted to use IRSA for creating the SA/permissions for cloudwatch agent. Unless we enable this, IRSA doesn't work